hyperlink/_url.py

```diff
@@ -1585,20 +1585,37 @@ def get(self, name):
         """
         return [value for (key, value) in self.query if name == key]
 
-    def remove(self, name):
+    def remove(self, name, value=_UNSET, limit=None):
         """Make a new :class:`URL` instance with all occurrences of the query
-        parameter *name* removed. No exception is raised if the
+        parameter *name* removed, or, if *value* is set, parameters
+        matching *name* and *value*. No exception is raised if the
         parameter is not already set.
 
         Args:
             name (unicode): The name of the query parameter to remove.
+            value (unicode): Optional value to additionally filter
+               on. Setting this removes query parameters which match
```

Comment:
If you could drop the word "all" in here, that would I think fully address my concern.

Reply:
done!